src/main/java/hudson/plugins/ec2/EC2RetentionStrategy.java

```diff
@@ -207,7 +207,19 @@ protected Object readResolve() {
     }
 
     public void taskAccepted(Executor executor, Queue.Task task) {
-        return;
+        EC2Computer computer = (EC2Computer) executor.getOwner();
+        EC2AbstractSlave slaveNode = computer.getNode();
+        int maxTotalUses = slaveNode.maxTotalUses;
+        if (maxTotalUses <= -1) {
+            LOGGER.info("maxTotalUses set to unlimited (" + slaveNode.maxTotalUses + ") for agent " + slaveNode.instanceId);
```

Comment:
I think it's debatable that this log message is of any use. At least at the `INFO` level.

Reply:
Adding bunch of those loggers basically helps during the tests... I don't mind removing it tho.
```
0.361 [id=111]       INFO    jenkins.InitReactorRunner$1#onAttained: Completed initialization
   0.369 [id=94]        INFO    h.p.ec2.EC2RetentionStrategy#taskAccepted: maxTotalUsesInstance has 3 builds left
   0.369 [id=94]        INFO    h.p.ec2.EC2RetentionStrategy#taskAccepted: maxTotalUsesInstance has 2 builds left
   0.370 [id=94]        INFO    h.p.ec2.EC2RetentionStrategy#taskAccepted: maxTotalUsesInstance has 1 builds left
   0.370 [id=94]        INFO    h.p.ec2.EC2RetentionStrategy#taskAccepted: maxTotalUses drained - suspending agent maxTotalUsesInstance
   0.370 [id=94]        INFO    h.p.ec2.EC2RetentionStrategy#postJobAction: Agent maxTotalUsesInstance is terminated due to maxTotalUses (1)
   0.370 [id=94]        INFO    h.p.ec2.EC2RetentionStrategy#taskAccepted: maxTotalUses drained - suspending agent maxTotalUsesInstance
   0.370 [id=94]        INFO    h.p.ec2.EC2RetentionStrategy#postJobAction: Agent maxTotalUsesInstance is terminated due to maxTotalUses (0)
   0.371 [id=94]        INFO    h.p.ec2.EC2RetentionStrategy#taskAccepted: maxTotalUses set to unlimited (-1) for agent maxTotalUsesInstance
   0.372 [id=94]        INFO    jenkins.model.Jenkins#cleanUp: Stopping Jenkins
```

Reply:
Right, I'd just adjust to `LOGGER.debug`.